core-ktlint/src/main/kotlin/com/twitter/rules/core/ktlint/KtlintComposeKtConfig.kt

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 Twitter, Inc.
+// SPDX-License-Identifier: Apache-2.0
+package com.twitter.rules.core.ktlint
+
+import com.pinterest.ktlint.core.api.EditorConfigProperties
+import com.twitter.rules.core.ComposeKtConfig
+import com.twitter.rules.core.util.toSnakeCase
+
+/**
+ * Manages the configuration for ktlint rules. In ktlint, configs are typically in snake case, while in the
+ * whole project and in detekt they are camel case, so this class will convert all camel case keys to snake case.
+ * Results will be memoized as well, as config shouldn't be changing during the lifetime of a rule.
+ */
+internal class KtlintComposeKtConfig(
+    private val properties: EditorConfigProperties
+) : ComposeKtConfig {
+    private val cache = mutableMapOf<String, Any?>()
+
+    @Suppress("UNCHECKED_CAST")
+    private fun <T : Any> getValueAsOrPut(key: String, value: () -> T?): T? =
+        cache.getOrPut(key) { value() } as? T
+
+    override fun getInt(key: String, default: Int): Int =
+        getValueAsOrPut(key) { properties[key.toSnakeCase()]?.getValueAs<String>()?.toInt() } ?: default
```

Comment:
whenever invoking `properties[...]` I'm using the snake case version (to be consistent with how editorconfigs are done)